share/governance.md

```diff
@@ -9,6 +9,11 @@
   * [Project Management Committee (PMC)](#project-management-committee-pmc)
     * [PMC Onboarding](#pmc-onboarding)
   * [Specialized Roles](#specialized-roles)
+  * [Subcommittees](#subcommittees)
+    * [Module committee](#module-committee)
+    * [Security committee](#security-committee)
+    * [Language committee](#language-committee)
+    * [Ecosystem committee](#ecosystem-committee)
```

Comment:
Should an `Infrastructure Subcommittee` also be a thing as there is bound to be more of that going forward?

Reply:
What would that entail? I thought each subcommittee would deal with the relevant infra such as Module's handling modulesync, Ecosystem handling packaging and repos, etc.

Reply:
I'd describe the  infrastructure subcommittee as for the packaging site, self-hosted runners, and any non-github based infrastructure that the project needs as well as managing mirroring and sponsorship around infrastructure.

We know we're going to need apt/yum/dnf/mac/windows package repositories that are not super easy to host in github, as well as more self-hosted runners for building and testing on various platforms.

Reply:
I'd argue GitHub itself is also infra. Today the PMC takes the admin role but if we're a larger org I don't see why a more dedicated infra team can't take ownership. We can discuss if there should be a strict boundary. Back in university I was part of an org where the rule was that the board wasn't doing operational infra things. Partly to keep focus and that argument may be irrelevant here but it can be

Reply:
Suggest that PMC take this on. If it is too much effort, they should start a subcommittee or delegate it to an existing one. We will have other things like this where we do not know the scope of the work and this allows them to handle it organically.